package/MDAnalysis/topology/guessers.py

```diff
@@ -475,3 +475,48 @@ def guess_atom_charge(atomname):
     """
     # TODO: do something slightly smarter, at least use name/element
     return 0.0
+
+
+def guess_aromaticities(atomgroup):
+    """Guess aromaticity of atoms using RDKit
+
+    Parameters
+    ----------
+    atomgroup : mda.core.groups.AtomGroup
+        Atoms for which the aromaticity will be guessed
+
+    Returns
+    -------
+    aromaticities : numpy.ndarray
+        Array of boolean values for the aromaticity of each atom
+
+    .. versionadded:: 2.0.0
+    """
+    mol = atomgroup.convert_to("RDKIT")
+    atoms = sorted(mol.GetAtoms(),
+                   key=lambda a: a.GetIntProp("_MDAnalysis_index"))
+    return np.array([atom.GetIsAromatic() for atom in atoms])
+
+
+def guess_gasteiger_charges(atomgroup):
```

Comment:
I think for now it's fine, although long term we might want to consider how we want this to look API-wise. In a hypothetical world where we have converters for things like openbabel, we might end up with multiple possible charge assignment methods. It'd be nice to have them all accessible from the same entry point.